platform/packaging/build/faiss/Dockerfile

```diff
@@ -80,12 +96,12 @@ RUN apt install -y cuda-toolkit-12-2
 RUN apt install -y intel-oneapi-mkl
 RUN apt install -y intel-oneapi-mkl-devel
 
-RUN curl -sLO https://github.com/facebookresearch/faiss/archive/refs/tags/v1.7.4.tar.gz
-RUN tar -xf v1.7.4.tar.gz
+RUN curl -L https://github.com/facebookresearch/faiss/archive/refs/tags/${FAISS_VERSION}.tar.gz -O /workspace/sources/faiss-${FAISS_VERSION}.tar.gz
+#RUN curl -sLO https://github.com/facebookresearch/faiss/archive/refs/tags/${FAISS_VERSION}.tar.gz
+RUN tar xf /workspace/sources/${FAISS_VERSION}.tar.gz --directory=/workspace/sources/faiss-${FAISS_VERSION} --strip-components=1
 
-WORKDIR /workspace/faiss-1.7.4/
-RUN patch -p1 --input=/workspace/build/faiss-1.7.4-runpath-origin.patch
-RUN patch -p1 --input=/workspace/build/faiss-1.7.4-setuppy-include-binaries.patch
+RUN patch --strip=1 --directory="/workspace/sources/faiss-${FAISS_VERSION}" --input="/workspace/patches/faiss-${FAISS_VERSION}-runpath-origin.patch"
+RUN patch --strip=1 --directory="/workspace/patches/faiss-${FAISS_VERSION}" --input="/workspace/patches/faiss-${FAISS_VERSION}-setuppy-include-binaries.patch"
```

Comment:
Directory should have "sources" instead of "patches".

Reply:
agree, thanks.